homeassistant/components/tuya/strings.json

```
@@ -231,6 +231,12 @@
       },
       "maximum_liquid_depth": {
         "name": "Maximum liquid depth"
+      },
+      "lower_temperature": {
+        "name": "Lower temperature"
+      },
+      "upper_temperature": {
+        "name": "Upper temperature"
       }
```

Are those target temperatures? In that case it might be a good idea to include that in the names:

```suggestion
      "lower_temperature": {
        "name": "Lower target temperature"
      },
      "upper_temperature": {
        "name": "Upper target temperature"
      }
```

That's how they are called in the `climate` integration.